packages/base/src/3dview/mainview.tsx

```diff
@@ -1064,6 +1071,7 @@ export class MainView extends React.Component<IProps, IStates> {
         boundingBox.position.copy(center);
 
         selectedMesh.add(boundingBox);
```

Comment:
Shall we not do this anymore? Now that you add it to the group I feel like we should not also add it to the mesh?

Reply:
Yes, that sounds good.